### DISCUSSION TITLE
Icons not showing up on Windows

### DISCUSSION THREAD

**Discussion by hxpe-dev** [helix-editor/helix#8534] [Troubleshooting] [answered]
Hi, i'm using windows and for some reason when I open a rust script I have an icon like this : [?] at the start of every line. It's the icon like the wanted icon was not found or something like that.
[Image showing the issue](https://ibb.co/0cyFnyJ)

**Comment by kirawi** [comment]
Probably something to do with `cmd.exe`'s default font or how it handles the character (this is fixed on Windows Terminal). Try a different font or a different terminal emulator.

**Reply by hxpe-dev** [reply]
How can I change the font?

**Reply by CptPotato** [reply, marked as answer]
> How can I change the font?

If you're running Cmd, right-click the title bar and go to `Properties`.
